client/components/checkout.js

```diff
@@ -1,68 +1,83 @@
         <div>
-            <h1>Checkout!</h1>
-            <h2>Your Order:</h2>
+          <h3>Address Information: (If you don't put in the right information, you don't get your products)</h3>
+          <label>
+            Street Address: 
+            <input 
+              name="street"
+              type="text"
+              onChange={props.handleChange}
+              required
+            />
+          </label>
+          <label>
+            City: 
+            <input 
+              name="city"
+              type="text"
+              onChange={props.handleChange}
+              required
+            />
+          </label>
+          <label>
+            State: 
+            <input 
+              name="state"
+              type="text"
+              onChange={props.handleChange}
+              required
+            />
+          </label>
+          <label>
+            ZIP code (only first 5): 
+            <input 
+              name="zip"
+              type="text"
+              onChange={props.handleChange}
+              required
+            />
+          </label>
         </div>
-        <form onSubmit={event => props.handleSubmit(event, props.address, props.userId, props.cartId)} >
-            <div>
-                <h3>Address Information: (If you don't put in the right information, you don't get your products)</h3>
-                <label>
-                  Street Address: 
-                  <input 
-                    name="street"
-                    type="text"
-                    onChange={props.handleChange}
-                    required
-                  />
-                </label>
-                <label>
-                  City: 
-                  <input 
-                    name="city"
-                    type="text"
-                    onChange={props.handleChange}
-                    required
-                  />
-                </label>
-                <label>
-                  State: 
-                  <input 
-                    name="state"
-                    type="text"
-                    onChange={props.handleChange}
-                    required
-                  />
-                </label>
-                <label>
-                  ZIP code (only first 5): 
-                  <input 
-                    name="zip"
-                    type="text"
-                    onChange={props.handleChange}
-                    required
-                  />
-                </label>
-            </div>
-            <div>
-                <h3>Payment Information:</h3>
-            </div>
-            <button
-              type="submit"
-            >Submit
-            </button>
-        </form>
+        <button
+          type="submit"
+          ref={input => { inputElement = input; }}
+        />
+      </form>
+      <h3>Payment Information:</h3>
+      <StripeCheckout
+        stripeKey="pk_test_vt2tbGVU5eFuxMOOljFfZHew"
```

Comment:
So I'm going to add the secret key to Heroku's config vars. I don't think it's good practice to put the secret in the code.

Reply:
https://hackernoon.com/how-to-use-environment-variables-keep-your-secret-keys-safe-secure-8b1a7877d69c

Reply:
I think that the normal thing to do is to have it in the .env file, and then you can access it with something like:
const stripe = require('stripe')(process.env.STRIPE_SECRET);

Reply:
ninja'd... rip

Reply:
I'm dumb this is the public key.